templates/base.html

```diff
@@ -224,6 +237,22 @@ <h1 class="modal-title fs-5" id="modalConfirmDeleteLabel"></h1>
     </script>
 
     {% block extra_scripts %} {% endblock %}
+
+    <!-- Visitor analytics -->
+    <script>
+        var _paq = window._paq = window._paq || [];
+        /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+        _paq.push(["disableCookies"]);
+        _paq.push(['trackPageView']);
+        _paq.push(['enableLinkTracking']);
+        (function () {
+            var u = "//matomo.dckube.scilifelab.se/";
+            _paq.push(['setTrackerUrl', u + 'matomo.php']);
+            _paq.push(['setSiteId', '5']);
+            var d = document, g = d.createElement('script'), s = d.getElementsByTagName('script')[0];
+            g.async = true; g.src = u + 'matomo.js'; s.parentNode.insertBefore(g, s);
+        })();
+    </script>
```

Comment:
Don't know how to verify that this works, so i'll just approve and hope for the best!